tests/runner.py

```diff
@@ -1029,6 +1029,24 @@ def log_request(code=0, size=0):
 
 def server_func(dir, q, port):
   class TestServerHandler(SimpleHTTPRequestHandler):
+    def send_head(self):
+      if self.path.endswith('.js'):
```

Comment:
btw, separately, it would be good to fix the wasm mimetype here (`application/wasm`), as without it we get a warning and don't test streaming compilation.